apps/entity-service/src/entities/dto/project.dto.ts

```diff
@@ -76,7 +76,7 @@ export class ProjectLightDto extends EntityDto {
   updatedAt: Date;
 
   @ApiProperty()
-  treesPlantedCount: number;
+  treesPlantedCount?: number;
```

Comment:
The convention in the DTOs is to make this `| null;` and use `{ nullable: true }` in the decorator.